src/main/resources/templates/index.html

```diff
@@ -26,8 +26,12 @@ <h2 th:text="${category.name}"/>
 				</section>
 				<ul class="items hidden">
 					<li class="item" th:each="product : ${category.products}">
-						<p th:text="${product.name}"/>
-						<button class="add" th:value="${product.id}">Add</button>
+						<h3 th:text="${product.name}"/>
+						<div th:id="'product-'+${product.id}" class="interface">
+							<button class="add" th:value="${product.id}">+</button>
+							<p th:text="${cart.getLineItemQuantityByProductId(product.id)}"></p>
```

Comment:
i think you can take this get out. unless its getget then my bad.

Reply:
I actually just made that getter for this purpose, it does what it says it does but if LineItem doesn't exist it returns 0.